README.md

```diff
@@ -303,6 +312,16 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Notes**:  See this page for their [FAQ's](https://substrate.dev/hacktoberfest/faq/) or this page for their [official announcement](https://www.parity.io/hacktoberfest-2019-parity/).
 
 
+### 4 Merged PRs
```

Comment:
No real need to make a new level for 4 merged PRs, you can just add it to the **2 or more merged PR** section

Reply:
Sure.